src/java.base/share/classes/java/lang/classfile/MethodSignature.java

```diff
@@ -52,7 +56,7 @@ public sealed interface MethodSignature
     String signatureString();
 
     /**
-     * @return method signature for a raw (no generic information) method descriptor
+     * {@return method signature for a raw (no generic information) method descriptor}
```

Comment:
Isn't this missing an article - e.g. `return the method signature` or `return a method signature` ?

Reply:
Fixed, thanks.